src/bootstrap/config.rs

```diff
@@ -663,6 +665,7 @@ impl Config {
         config.python = build.python.map(PathBuf::from);
         set(&mut config.low_priority, build.low_priority);
         set(&mut config.compiler_docs, build.compiler_docs);
+        set(&mut config.docs_minification, build.docs_minification);
```

Comment:
Are you sure this gets set to `true` if the option isn't set?

Reply:
Oh, good catch. Yeah, seems like that can be adjusted.